core/domain/classifier_services.py

```diff
@@ -442,7 +445,17 @@ def load_examples(self, examples):
             xrange(self._num_docs))
 
     def predict_label_for_doc(self, d):
-        """Returns the predicted label from a doc's prediction report."""
+        """Returns the predicted label from a doc's prediction report.
+
+        Classifiers built with less than _DEFAULT_MIN_DOCS_TO_PREDICT will
```

Comment:
This comment has been addressed and is superseded by: https://github.com/oppia/oppia/pull/1059/files#r45274104